MatrixKit/Views/RoomBubbleList/MXKRoomBubbleTableViewCell.m

```diff
@@ -409,6 +488,12 @@ - (void)render:(MXKCellData *)cellData
                 longPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(onLongPressGesture:)];
                 [self.attachmentView addGestureRecognizer:longPress];
             }
+            
+            // Handle auto animation of the gif (if the option is turned on)
+            if (_isAutoAnimatedGif)
+            {
+                self.isAutoAnimatedGif = YES;
```

Comment:
It looks a bit weird to set again the same value in order to get the right display. Hence my proposal for `renderGIF` or an equivalent method?